documentation/docs/guidedtour.md

```diff
@@ -0,0 +1,183 @@
+   </p> 
+   
+1. Provide a name for your new item (for example, *My First Pipeline*) and select **Pipeline**.
+   <p align="center">
+   ![Create Pipeline Job](../images/JenkinsNewItemPipeline-1.png "Jenkins New Item")
+   </p>  
+
+1. For **Definition** in the **Pipeline** options, choose **Pipeline script from SCM**. 
+
+1. For **SCM**, choose **Git**.
+
+1. For **Repository URL** in the **Repositories** section, enter the URL of your Git repository, for example `https://github.com/<your-org>/cloud-cf-helloworld-nodejs`. **Note:** If your repository is protected, you must provide your credentials in the **Credentials** section.
+   <p align="center">
+   ![Create Pipeline Job](../images/JenkinsNewItemPipeline-2.png "Jenkins New Item")
+   </p>  
+
+1. Choose **Save**. 
+
+1. To run your pipeline, choose **Build Now** in the job UI.
+
+
+## Add a Build Step
+
+1. In your `Jenkinsfile`, add the following code snippet: 
+   ```
+    stage('build') {
+      mtaBuild script: this
+    }
+   ```
+   **Result:** The `mtaBuild` step calls a build tool to build a multi-target application (MTA). The tool consumes an MTA descriptor that contains the metadata of all entities which comprise an application or are used by one during deployment or runtime, and the dependencies between them. For more information about MTAs, see [sap.com][sap]. 
+   
+2. Create an MTA descriptor with the name `mta.yaml`, which contains the following code:
+   ```
+    _schema-version: 2.1.0
+    ID: com.sap.piper.node.hello.world
+    version: 1.0.0
+    description: A Hello World sample application
+    provider: SAP Sample generator
+    modules:
+      - name: piper.node.hello.world
+        type: nodejs
+        path: .
+   ```
+   
+3. To configure the step to build an MTA for the Cloud Foundry environment, in your repository, open or create the `.pipeline/config.yml` and add the following content: 
+   ```
+    general:
+    steps:
+      mtaBuild:
+        buildTarget: 'CF'
+   ```
+   For additional information about the configuration, have a look at the [Common Configuration Guide][resources-configuration] and the [MTA build step documentation][resources-step-mtabuild].
+
+4. Commit your changes.
+
+5. To run your pipeline, choose **Build Now** in the job UI.
+
+## Add a Deploy Step
+
+1.  In your `Jenkinsfile`, add the following code snippet:
+   ```
+   stage('deploy') {
+     def mtarFilePath = commonPipelineEnvironment.getMtarFilePath()
+     cloudFoundryDeploy( script: this, mtaPath: mtarFilePath)
+   }
+   ```
+   **Result:** The `cloudFoundryDeploy`  step calls the Cloud Foundry command line client to deploy into SAP Cloud Platform.
+
+2. To configure the step to deploy into the Cloud Foundry environment, in your repository, open or create the `.pipeline/config.yml` and add the following content:
+   ```
+    cloudFoundryDeploy:
+      deployTool: 'mtaDeployPlugin'
+      deployType: 'standard'
+      cloudFoundry:
+        org: '<your-organisation>'
```

Comment:
❗️ Here we propose to expose the org and the space. The way how the guide is setup we are working on github.com (see above: 

> Fork the cloud-cf-helloworld-nodejs repository into your GitHub organization.

To my knowledge we can only for inside one github instance (correct me if I'm wrong: is it also possible to fork from github.com to a github enterprise instance).

This we must not do IMO.

I know org and space are not passwords in the narrower sense, but they open a attack vector.